src/appshell/view/navigableappmenumodel.cpp

```diff
@@ -277,11 +277,13 @@ bool NavigableAppMenuModel::processEventForAppMenu(QEvent* event)
                                   && isSingleSymbol
                                   && isNavigationStarted;
     bool isNavigationWithAlt = (modifiers & Qt::AltModifier)
+                               && !(modifiers & Qt::ControlModifier)
                                && !(modifiers & Qt::ShiftModifier)
                                && isSingleSymbol;
 
     bool isAltKey = key == Qt::Key_Alt
                     && key != Qt::Key_Shift
```

Comment:
Strictly speaking unrelated, but the `&& key != Qt::Key_Shift` seems redundant here